charts/rasa-x/templates/_rasa.tpl

```diff
@@ -1,17 +1,17 @@
 {{/* Return Rasa OSS URL - the production environment */}}
 {{- define "rasa.production.url" -}}
-{{- if and (not .Values.rasa.versions.rasaProduction.enabled) .Values.rasa.versions.rasaProduction.external.enabled (not (empty .Values.rasa.versions.rasaProduction.external.url)) (not ((index .Values "rasa-bot").subchart)) }}
+{{- if and (not .Values.rasa.versions.rasaProduction.enabled) .Values.rasa.versions.rasaProduction.external.enabled (not (empty .Values.rasa.versions.rasaProduction.external.url)) (not ((index .Values "rasa").subchart)) }}
```

Comment:
for now, it should stay as it is ("rasa-bot")

Reply:
Sounds good. I close the PR then?